app/src/main/java/org/mozilla/fenix/compose/list/ExpandableListHeader.kt

```diff
@@ -36,45 +36,54 @@ import org.mozilla.fenix.theme.FirefoxTheme
  * @param expandActionContentDescription The content description for expanding the section.
  * @param collapseActionContentDescription The content description for collapsing the section.
  * @param onClick Optional lambda for handling header clicks.
+ * @param subContent Optional Composable for adding UI to the end of the header.
  */
+@Suppress("LongParameterList")
 @Composable
 fun ExpandableListHeader(
     headerText: String,
     expanded: Boolean? = null,
     expandActionContentDescription: String? = null,
     collapseActionContentDescription: String? = null,
     onClick: () -> Unit = {},
+    subContent: @Composable () -> Unit = {},
```

Comment:
Let's rename this to `actions`. I think we should name the parameter accordingly to the expected 'content' that would be added at the end of the header for this component instead of a more generic name like `subContent`, which you would expect to see for something like `Button`.